recipe/meta.yaml

```diff
@@ -146,5 +146,6 @@ about:
 
 extra:
   recipe-maintainers:
+    - mxr-conda
```

Comment:
@rluria14
Aside from no further comment or previous involvement on the feedstock (which would still be OK for someone with as many feedstocks under their belt), I'm surprised that you're adding another user?

I now looked at the profile of @mxr-conda - can you explain the intention of adding:
> Team account for following updates to conda-forge maintained feedstocks

Updates to any feedstock can already be followed through the github UI ("Watch" -> "All Activity"), so I don't understand why this account needs to be a maintainer?

Reply:
@h-vetinari myself and @rluria14 work collaboratively on most feedstocks we maintain, along with another maintainer on conda-forge. we created a team account to better maintain all the 300+ feedstocks we collaboratively maintain, which is the maintainer you see here.

Reply:
@h-vetinari I see how that description can lead you astray. I've updated it to the following: `Team account for maintaining jointly contributed conda-forge feedstocks.`

Reply:
OK thanks for the info. Could you add something along the lines of:
> "this is a shared account by @oblute @rluria14 & xxx for jointly managing conda feedstocks"

to the description of @mxr-conda? I don't immediately understand the need (but then, I'm not maintaining nearly as many feedstocks as you are 😅), and I'm mainly worried about security by adding a - superficially - more or less random account. By having that transparency, it would at least point to being run by seasoned conda-forged maintainers.

Reply:
No problem! Went ahead and included our github handles in the description. Thanks

Reply:
Thanks!

Hope that the shared account brings some benefits for y'all and thanks for all the work you do in maintaining packages! 😊